### ISSUE TITLE
Banish Reduction's Inevitable Escalation to bonus missions for unconventional gameplay?

### ISSUE THREAD

**Issue by WhenTanksFly** [ReactiveDrop/reactivedrop_public_src#729]
Last time we had an unorthodox mission in an addon that was going to be merged into the base game, the entire gimmick of invisible enemies and starting without guns was removed to fit it into normal gameplay.
Now your horde shooter has a mission with no aliens nor guns. What a chore.

Reduction is generally regarded as one of the least polished campaigns; many of its objectives have triggers so unresponsive that it feels like the host is regularly catching 10s screen freezes. Extra 2 bars of frustration added every time the progress arbitrarily stops just to give the players time to read the now-absent dialogue.

The flow disruption peaks at Inevitable Escalation. Good luck not falling asleep doing all objectives yourself in the _average public lobby™_, or calling tech and welder stealers for minutes. Well, at least I don't risk my marine's life alt-tabbing to Google Translate. Then someone blocks the nuke 1cm away from its finish position. I fear the flame coming from my butt may combust the bomb.

All this insane action does not belong in vanilla environment. Not like it rewards anything. Zero replayability as well.
(What remains of) the campaign's storytelling wouldn't be hurt much - the flashback mission messes it up a lot more even in the Workshop version with dialogue intact.

Could also get the original Invisible Threat into bonus missions while we're at it. Can't be worse than moving a package around a facility.
Or you could add Infinite Inevitable Escalation as a nuclear middle finger. Who can stand it the longest??

**tl;dr:** title
